packages/fxa-content-server/tests/functional/sign_in_token_code.js

```diff
@@ -103,6 +105,22 @@ registerSuite('signin token code', {
       );
     },
 
+    'verified - treatment-code - bounce': function() {
+      experimentParams.query.forceExperiment = 'tokenCode';
+      experimentParams.query.forceExperimentGroup = 'treatment-code';
+
+      return this.remote
+        .then(openFxaFromRp('enter-email', experimentParams))
+        .then(fillOutEmailFirstSignIn(email, PASSWORD))
+        .then(testElementExists(selectors.SIGNIN_TOKEN_CODE.HEADER))
+        .then(getStoredAccountByEmail(email))
+        .then(destroySessionForEmail(email))
```

Comment:
While looking at https://github.com/mozilla/fxa/pull/3533/files, why would this have to use `destroySessionForEmail` verses `accountDestroy`?

Reply:
A sign in verification email doesn't normally destroy the account, that only happens on signup verifications.